sycl/test/basic_tests/min_max_test.cpp

```diff
@@ -0,0 +1,9 @@
+#include "windows.h"
```

Comment:
This test will fail on linux. Please add something like `// REQUIRES: windows` (take a look to other tests to find exact spelling for `windows` feature)

Reply:
And RUN line is missing.